static/css/rue89.css

```diff
@@ -0,0 +1,134 @@
+a:link,
```

Comment:
@mperrien My understanding is that the bundles are compiled from the SCSS files.  So wouldn't it be a lot easier to modify the SCSS source files directly instead of leaving them as they are and then overriding the settings in a CSS file?

I'm worried that with the current file it will be next to impossible to rebase the change on top of any upstream changes, whereas small diffs directly inside the SCSS files seem easier to maintain.

It would be great if you could remove the cache files.